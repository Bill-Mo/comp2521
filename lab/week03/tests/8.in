13 8 6 18 7 17 15
