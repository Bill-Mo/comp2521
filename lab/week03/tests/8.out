Tree:
     13
     / \
    /   \
   /     \
  8      18
 /       /
6       17
 \     /
  7   15

#nodes = 7,  #leaves = 2
Infix  : 6 7 8 13 15 17 18 
Prefix : 13 8 6 7 18 17 15 
Postfix: 7 6 8 15 17 18 13 
ByLevel: 13 8 18 6 17 7 15 
